DownUnder.UI/UI/Widgets/DataTypes/SerializableType.cs

```diff
@@ -1,114 +1,60 @@
-        {
-            get
-            {
-                if (type == null)
-                    return null;
-                return type.FullName;
-            }
-            set
-            {
-                if (value == null)
-                    type = null;
-                else
-                {
-                    type = Type.GetType(value);
-                }
-            }
+        string TypeString {
+            get => type?.FullName;
+            set => type = value == null ? null : Type.GetType(value);
         }
 
         // constructors
-        public SerializableType()
-        {
-            type = null;
+        public SerializableType() {
         }
-        public SerializableType(Type t)
-        {
+
+        public SerializableType(Type t) {
             type = t;
         }
 
         // allow SerializableType to implicitly be converted to and from System.Type
-        static public implicit operator Type(SerializableType stype)
-        {
-            return stype.type;
-        }
-        static public implicit operator SerializableType(Type t)
-        {
-            return new SerializableType(t);
-        }
+        public static implicit operator Type(SerializableType stype) => stype.type;
+        public static implicit operator SerializableType(Type t) => new SerializableType(t);
 
         // overload the == and != operators
-        public static bool operator ==(SerializableType a, SerializableType b)
-        {
+        public static bool operator ==(SerializableType a, SerializableType b) {
             // If both are null, or both are same instance, return true.
-            if (System.Object.ReferenceEquals(a, b))
-            {
+            if (ReferenceEquals(a, b))
                 return true;
-            }
 
             // If one is null, but not both, return false.
-            if (((object)a == null) || ((object)b == null))
-            {
+            if (a == null || b == null)
                 return false;
-            }
 
             // Return true if the fields match:
             return a.type == b.type;
         }
-        public static bool operator !=(SerializableType a, SerializableType b)
-        {
-            return !(a == b);
-        }
-        // we don't need to overload operators between SerializableType and System.Type because we already enabled them to implicitly convert
 
-        public override int GetHashCode()
-        {
-            return type.GetHashCode();
-        }
+        public static bool operator !=(SerializableType a, SerializableType b) => !(a == b);
 
-        // overload the .Equals method
-        public override bool Equals(System.Object obj)
-        {
-            // If parameter is null return false.
-            if (obj == null)
-            {
-                return false;
-            }
+        // we don't need to overload operators between SerializableType and System.Type because we already enabled them to implicitly convert
```

Comment:
I take it as a good sign that my most criticized code yet is a copy paste from stackoverflow, tons of good changes